geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/TestWithFeatureType.scala

```diff
@@ -0,0 +1,38 @@
+/***********************************************************************
+ * Copyright (c) 2013-2020 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.accumulo
+
+import org.geotools.data.Query
+import org.geotools.filter.text.ecql.ECQL
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+
+/**
+ * Trait to simplify tests that require reading and writing features from an AccumuloDataStore
+ */
+trait TestWithFeatureType extends TestWithDataStore {
```

Comment:
If we factor out building up a common collection of test features, this code may be on the list to share, etc.

Reply:
:+1: